code/modules/butchery/butchery_products.dm

```diff
@@ -16,7 +16,7 @@
 	var/fat_material    = /decl/material/solid/organic/meat/gut
 	var/meat_name       = "meat"
 
-/obj/item/food/butchery/Initialize(ml, material_key, mob/living/donor)
+/obj/item/food/butchery/Initialize(mapload, material_key, skip_plate = FALSE, mob/living/donor)
```

Comment:
are you sure the `= FALSE` doesn't make it get treated as a keyword argument? is there a reason it's all `= FALSE` instead of just letting it be null?